Resources/config/schema/redis-1.0.xsd

```diff
@@ -72,6 +73,15 @@
         </xsd:restriction>
     </xsd:simpleType>
 
+    <xsd:simpleType name="phpredis-slave-failover-type">
```

Comment:
I did not know about this file :open_mouth: 

@curry684 Should we continue to maintain it? Is this an old best practice for bundles?

Reply:
Can't say I've ever seen it either haha. Wasn't even aware that Symfony had XSD support for this, I was always under the assumption that the TreeBuilder had been around for that since ages past.

I think it's a relic from the really old days, before config went to YAML and now PHP itself. It's likely immensely outdated as well.

Reply:
Thank you :-)